app/assets/stylesheets/geoblacklight.scss

```diff
@@ -1,3 +1,3 @@
 @import 'bootstrap/scss/bootstrap';
-@import 'blacklight-frontend/app/assets/stylesheets/blacklight/blacklight';
+@import 'blacklight/blacklight';
```

Comment:
is it ok to commit this change and the `bin/build-css` file? (not questioning it so much as verifying it's ok)